packages/alignments/src/CramAdapter/CramSlightlyLazyFeature.ts

```diff
@@ -246,8 +340,9 @@ export default class CramSlightlyLazyFeature implements Feature {
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     const tags: Record<string, any> = {}
     this.tags().forEach((t: string) => {
-      tags[t] = this.get(t)
+      tags[t] = this._get(t)
```

Comment:
This preserves case for any extra CRAM "tags" (not actually tags, they are things like CIGAR)